drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -172,6 +172,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     block_settings_key = 'drag-and-drop-v2'
     has_score = True
 
+    def max_score(self):  # pylint: disable=no-self-use
+        """
+        Return the problem's max score, which for DnDv2 always equals 1.
```

Comment:
The formula intentionally puts the `weight` in both the numerator and the denominator.  The reason is because the `weighted_possible` (denominator) needs to be the total possible points, including the `weight`.  And therefore, the `weighted_earned` (numerator) is also multiplied by the `weight` in order to keep the total score the same.